endpoints.json

```diff
@@ -43,5 +43,99 @@
         }
       ]
     }
+  },
+  "GET /api/articles/:article_id/comments": {
+    "description": "serves an array of comments from a single article",
+    "queries": ["article_id", "comments"],
+    "exampleResponse": [
+      {
+        "comment_id": 5,
+        "votes": 0,
+        "created_at": "2020-11-03T21:00:00.000Z",
+        "author": "icellusedkars",
+        "body": "I hate streaming noses",
+        "article_id": 1
+      },
+      {
+        "comment_id": 2,
+        "votes": 14,
+        "created_at": "2020-10-31T03:03:00.000Z",
+        "author": "butter_bridge",
+        "body": "The beautiful thing about treasure is that it exists. Got to find out what kind of sheets these are; not cotton, not rayon, silky.",
+        "article_id": 1
+      },
+      {
+        "comment_id": 18,
+        "votes": 16,
+        "created_at": "2020-07-21T00:20:00.000Z",
+        "author": "butter_bridge",
+        "body": "This morning, I showered for nine minutes.",
+        "article_id": 1
+      },
+      {
+        "comment_id": 13,
+        "votes": 0,
+        "created_at": "2020-06-15T10:25:00.000Z",
+        "author": "icellusedkars",
+        "body": "Fruit pastilles",
+        "article_id": 1
+      },
+      {
+        "comment_id": 7,
+        "votes": 0,
+        "created_at": "2020-05-15T20:19:00.000Z",
+        "author": "icellusedkars",
+        "body": "Lobster pot",
+        "article_id": 1
+      },
+      {
+        "comment_id": 8,
+        "votes": 0,
+        "created_at": "2020-04-14T20:19:00.000Z",
+        "author": "icellusedkars",
+        "body": "Delicious crackerbreads",
+        "article_id": 1
+      },
+      {
+        "comment_id": 6,
+        "votes": 0,
+        "created_at": "2020-04-11T21:02:00.000Z",
+        "author": "icellusedkars",
+        "body": "I hate streaming eyes even more",
+        "article_id": 1
+      },
+      {
+        "comment_id": 12,
+        "votes": 0,
+        "created_at": "2020-03-02T07:10:00.000Z",
+        "author": "icellusedkars",
+        "body": "Massive intercranial brain haemorrhage",
+        "article_id": 1
+      },
+      {
+        "comment_id": 3,
+        "votes": 100,
+        "created_at": "2020-03-01T01:13:00.000Z",
+        "author": "icellusedkars",
+        "body": "Replacing the quiet elegance of the dark suit and tie with the casual indifference of these muted earth tones is a form of fashion suicide, but, uh, call me crazy — onyou it works.",
+        "article_id": 1
+      },
+      {
+        "comment_id": 4,
+        "votes": -100,
+        "created_at": "2020-02-23T12:01:00.000Z",
+        "author": "icellusedkars",
+        "body": " I carry a log — yes. Is it funny to you? It is not to me.",
+        "article_id": 1
+      },
+      {
+        "comment_id": 9,
+        "votes": 0,
+        "created_at": "2020-01-01T03:08:00.000Z",
+        "author": "icellusedkars",
+        "body": "Superficially charming",
+        "article_id": 1
+      }
```

Comment:
Instead of giving the whole response here just give one as an example - it's just a reference as to what they look like